subsys/lorawan/services/frag_transport.c

```diff
@@ -101,6 +101,9 @@ static struct frag_transport_context ctx;
 /* Callback for notification of finished firmware transfer */
 static void (*finished_cb)(void);
 
+/* Callback to handle descriptor field */
+static int (*handle_descriptor)(uint32_t descriptor);
```

Comment:
For consistency with the callback above:

```suggestion
static int (*descriptor_cb)(uint32_t descriptor);
```

Reply:
Looks like you accidentally dropped this line, which is why twister is now failing.

Reply:
Thanks for this. While I was doing the pushed, just noticed that I needed to do a rebase and in that process forgot to rebuild again.